infrastructure/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
 
   openhim-core:
     container_name: openhim-core
-    image: jembi/openhim-core:latest
+    image: openhim-alpinev3-multi-staged
```

Comment:
This is not a valid image. Please revert

Reply:
sure

Reply:
This has not been reverted

Reply:
Please check new commit